src/components/TextField/tests/TextField.test.tsx

```diff
@@ -694,11 +694,13 @@ describe('<TextField />', () => {
         const origialAddEventListener = document.addEventListener;
 
         beforeAll(() => {
-          document.addEventListener = jest.fn(
-            (eventType: string, callback: EventCallback) => {
-              documentEvent[eventType] = callback;
-            },
-          );
+          jest
```

Comment:
^

Reply:
Have changed the reassigning to use mockRestore().